packages/web-react/src/components/Pill/Pill.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { ActionColors, EmotionColors } from '../../constants';
+import { Pill } from './Pill';
+import ReadMe from './README.md';
+
+const meta: Meta<typeof Pill> = {
+  title: 'Components/Pill',
+  component: Pill,
+  parameters: {
+    docs: {
+      page: () => <Markdown>{ReadMe}</Markdown>,
+    },
+  },
+  argTypes: {
+    children: {
+      control: 'text',
+    },
+    color: {
+      control: 'select',
+      options: [...Object.values(ActionColors), ...Object.values(EmotionColors), 'selected', 'unselected'],
+      table: {
+        defaultValue: { summary: 'selected' },
+      },
+    },
+    elementType: {
+      control: 'text',
```

Comment:
When I try to use the control, an error occurs and I cannot go back:

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/c4ad2cd9-3ce4-4360-8c4f-f32e11ca564a)

Reply:
This happens when the element type value is empty. I added a default value to the control column, but it still crashes when you clear the field. But that is IMHO intended. Also, there is a button at the upper left with reload.